pkgs/applications/misc/deadd-notification-center/default.nix

```diff
@@ -1,62 +1,61 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, autoPatchelfHook
-, wrapGAppsHook
-, hicolor-icon-theme
-, gtk3
-, gobject-introspection
-, libxml2
-, fetchpatch
-}:
-stdenv.mkDerivation rec {
+{ mkDerivation, haskellPackages, fetchFromGitHub, lib, writeText }:
+
+let
+  systemd-service
+    = writeText "deadd-notification-center.service" ''
+        [Unit]
+        Description=Deadd Notification Center
+        PartOf=graphical-session.target
+
+        [Service]
+        Type=dbus
+        BusName=org.freedesktop.Notifications
+        ExecStart=$out/bin/deadd-notification-center
+
+        [Install]
+        WantedBy=graphical-session.target
+      '';
```

Comment:
We could just echo that into the final location because creating a derivation and then cp'ing it is no necessary.